src/chrome/local-runtime.ts

```diff
@@ -197,6 +205,36 @@ export default class LocalRuntime {
     }
   }
 
+  private async history(): Promise<NavigationHistory> {
+    const {Page} = this.client
+    await Promise.all([Page.enable()])
+    return await Page.getNavigationHistory()
+  }
+
+  private async back(): Promise<void> {
+    const history = await this.history()
+    if (history && history.currentIndex > 0) {
+      const {Page} = this.client
+      await Page.navigateToHistoryEntry({entryId: history.entries[history.currentIndex - 1].id})
+      await Page.loadEventFired()
```

Comment:
Uhm, no, it does not. You're right.

It also doesn't get triggered when you navigate from blah.html to blah.html#somewhere, which means the current goto-function is buggy, too - if I'm not mistaken.

Guess I just evaluate `window.history.back()`, etc. instead. Should have done that in the first place...